u4-databases/views/uni-iteration-details.hbs

```diff
@@ -0,0 +1,13 @@
+{{#>uni iteration}}
```

Comment:
`uni` include ไม่ต้องการ `iteration`
The `uni` include doesn't need `iteration`